api/src/wfl/service/slack.clj

```diff
@@ -57,7 +57,8 @@
 (def notifier (agent (PersistentQueue/EMPTY)))
 (add-watch notifier :watcher
            (fn [_key _ref _old-state new-state]
-             (log/debug (format "the current notification queue is: %s" (seq new-state)))))
+             (when (seq new-state)
```

Comment:
This emitted a log every second with the following message:

"the current notification queue is: nil"

Even for DEBUG level, did not seem useful, so only emit if there are any notifications in queue.